config/default.schema.json

```diff
@@ -238,12 +238,12 @@
 				},
 				"SCHOOL_SETTINGS_VISIBLE": {
 					"type": "boolean",
-					"default": false,
+					"default": true,
```

Comment:
Please do not chance this 2 default values.

Reply:
True, this change doesnt belong to this ticket issue, i fix in a new commit.